live-examples/html-examples/strong.html

```diff
@@ -0,0 +1 @@
+<p>... the most important rule, the rule you can never forget, no matter how much he cries, no matter how much he begs: <strong>never feed him after midnight</strong>.</p>
```

Comment:
Here as well, wondering is it makes sense to break onto two lines to avoid horizontal scrolling.

Reply:
Same answer as here: https://github.com/mdn/interactive-examples/pull/697#discussion_r181115779

Of course if making the editor wrap is not practical/sensible, we can break manually.